src/components/SignInButton.jsx

```diff
@@ -6,8 +6,7 @@ const SignInButton = () => {
     const provider = new GoogleAuthProvider();
     return signInWithPopup(auth, provider);
   };
-
-  return <button onClick={userSignIn}>Sign In</button>;
+  return <button onClick={userSignIn}>Continue With Google</button>;
 };
 
 export default SignInButton;
```

Comment:
This is the same as components/GoogleSignIn.jsx. Rename this to GoogleSignIn.jsx and remove components/GoogleSignIn.jsx. I don't want to put things in routes folder which are not related to route changes